### PR TITLE
Get Travis stuff into prod

### PR DESCRIPTION
**Pull request by briaguya**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# We will need these services:
+services:
+- docker
+- postgresql
+
+sudo: required
+
+# We need PostgreSQL 10 installed
+addons:
+  postgresql: "10"
```
Comment (ifsimicoded) [inline .travis.yml]: is this ok, given the change to 9.6 in your other branch?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  apt:
+    packages:
+      - postgresql-10
```
Comment (ifsimicoded) [inline .travis.yml]: same here -- postgres 9.6?

File: .travis.yml (hunk 1 above)
Comment (sergiogcx) [reply .travis.yml]: I have fixed this in my branch, we will continue to use 10. A specific step was added in the dockerfile to install Pg10.

Comment (sergiogcx) [reply .travis.yml]: I working to get this sorted before Thursday

**Pull request merged**